lamindb/core/loaders.py

```diff
@@ -156,7 +158,8 @@ def load_rds(path: UPathStr) -> UPathStr:
     ".parquet": pd.read_parquet,
     ".parquet.gz": pd.read_parquet,  # this doesn't work for externally gzipped files, REMOVE LATER
     ".fcs": load_fcs,
-    ".zarr": load_anndata_zarr,
+    ".zarr": load_zarr,
+    ".spatialdata.zarr": load_zarr,
```

Comment:
Why do we need `.spatialdata.zarr` here since `.zarr` uses the same reader?

Reply:
Hmm, using `.spatialdata.zarr` is more explicit but sure - in this case we can omit it :) Done.